NatLabRockies/tasty
# Feat/build composite shapes

PR to compose and wrap multiple "Simple Shapes" into "Composite Shapes" via the ShapesWrapper class.  If `type` is not provided it is assigned as a `point` base class for now - with inferred typing coming shortly.  

To Test: 

- Run the NREL VAV Cooling Only jupyter notebook example for Composite Shape construction and confirm validation.
